doc/lightning-ping.7.md

```diff
@@ -33,7 +33,7 @@ RETURN VALUE
 
 On success, the command will return an object with a single string.
 
-- *totlen*: A string that represents the answer length of the reply message (including header)
+- *totlen*: The length of the reply message (including header)
```

Comment:
```suggestion
- *totlen*: A string, length of the reply message (including header)
```

Reply:
This is not a string. See example below.

Reply:
```suggestion
- *totlen*: Integer. The length of the reply message (including header)
```